buildpack-deps/README.md

```diff
@@ -53,7 +53,7 @@ You are invited to contribute new features, fixes, or updates, large or small;
 we are always thrilled to receive pull requests, and do our best to process them
 as fast as we can.
 
-Before you start to code, we recommend discussing your plans 
+Before you start to code, we recommend discussing your plans
```

Comment:
This is autogenerated (as are most of the `*/README.md` files).  To fix this properly (and not have it come back), we need to change the template where this is generated (ie, remove the "mailing list" logic and have the one or two repos using it just use an entirely custom "user feedback" section instead).